docs/starters.yml

```diff
@@ -2383,3 +2383,16 @@
     - Uses MDX with Gatsby theme for quick and easy set up
     - Material-ui design with optional config passed into the theme options
     - Gradient background with sitemap, rss feed, and offline capabilities
+- url: https://github.com/gperl27/Gatsby-Starter-Blog-Typescript
+  repo: https://github.com/gperl27/Gatsby-Starter-Blog-Typescript
+  description: Gatsby starter blog with typescript
+  tags:
+    - Blog
+    - Typescript
+    - Styled Components
```

Comment:
Could you change this to `Styling:CSS-in-JS`. Tags should match categories on the [starter page](https://www.gatsbyjs.org/starters/?v=2).

Reply:
Updated